### PR TITLE
Solved #20 and #6

### LINKED ISSUE #6: Helpfile
Show helpfile in plugin.

### LINKED ISSUE #20: Distance of two profiles
The distance of two profiles seems to be exported as difference between x or y values, dependend on the selection in the options. The correct determination of the distances would be the distance between the points d = √((x2-x1)^2 )+ (y2-y1)^2)

### PR DESCRIPTION
**Pull request by MoritzMennenga**
Solved problem with the distances between profiles in #20 
Added the manual to the menu #6

### CODE REVIEW

File: example_data/corings.csv [hunk 1]
```diff
@@ -1,3 +1,4 @@
-wkt_geom;ID;loc_id;xcoord;ycoord;zcoorddb
-Point (32485574.71000000089406967 5949969.473000000230968);Core 1;1227;32485574.71;5949969.473;1.4596
-Point (32485583.17999999970197678 5950059.70899999979883432);Core 2;1226;32485583.18;5950059.709;1.3083
+ID;loc_id;xcoord;ycoord;zcoorddb
```
Comment (gricklom) [inline example_data/corings.csv:1]: Did you remove the column **wkt_geom** on purpose?

Comment (MoritzMennenga) [reply example_data/corings.csv:1]: It's unnecessary, thats why I removed it ( ... loc_id would be unnecessary, too). But it won't be a problem to let it in the file.

File: example_data/corings.csv [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
+Core 1;1227;32485574.71;5949969.473;1.4596
+Core 2;1226;32485583.18;5950059.709;1.3083
+Core 3;1226;32485603.18;5950090.709;1.8
```
Comment (gricklom) [inline example_data/corings.csv:4]: Is it correct, that Core 2 and Core 3 share the same loc_id?

File: example_data/corings.csv (hunk 1 above)
Comment (gricklom) [inline example_data/corings.csv:1]: Did you remove the column wkt_geom on purpose?

**Review by gricklom — APPROVED** [review]

File: example_data/corings.csv (hunk 2 above)
Comment (MoritzMennenga) [reply example_data/corings.csv:4]: I forgot to delete this column. It's not necessary and can be confusing in the expamle dataset.

**Pull request merged**